data/blog/new-website.md

```diff
@@ -0,0 +1,191 @@
+website to market and distribute their app.
+
+Using just a 
+[PNG icon file and a small amount of app metadata](https://github.com/electron/electron-apps/blob/master/contributing.md),
+we're able to collect a lot of information about a given app.
+Using data collected from GitHub, app pages can now display screenshots, 
+download links, versions, release notes, and READMEs for every app that 
+has a public repository. Using a color palette extracted from each app's icon,
+ we're able to produce [bold and accessible colors](https://github.com/zeke/pick-a-good-color)
+to give each app page some visual distinction.
+
+The [apps index page](https://electronjs.org/apps) now also has categories,
+and a keyword filter to find interesting apps like [GraphQL GUIs](https://electronjs.org/apps?q=graphql)
+and [p2p tools](https://electronjs.org/apps?q=graphql).
+
+If you've got an Electron app that you'd like featured on the site, open a 
+pull request on the [electron/electron-apps] repository.
+
+## One-line Installation with Homebrew
+
+The [Homebrew] package manager for macOS has a subcommand called [cask]
+that makes it easy to install desktop apps using a single command in your
+terminal, like `brew cask install atom`.
+
+We've begun collecting Homebrew cask names for popular Electron apps and are now
+displaying the installation command (for macOS visitors) on every app page 
+that has a cask:
+
+<figure>
+  <a href="https://electronjs.org/apps/dat">
+   <img class="screenshot" src="https://user-images.githubusercontent.com/2289/32871246-c5ef6f2a-ca34-11e7-8eb4-3a5b93b91007.png">
+   <figcaption>Installation options tailored for your platform: macOS, Windows, Linux</figcaption>
+  </a>
+</figure>
+
+To view all the apps that have homebrew cask names, visit 
+[electronjs.org/apps?q=homebrew](https://electronjs.org/apps?q=homebrew). If 
+you know of other apps with casks that we haven't indexed yet, 
+[please add them!](https://github.com/electron/electron-apps/blob/master/contributing.md)
+
+## 🌐 A New Domain
+
+We've moved the site from electron.atom.io to a new domain: [electronjs.org].
+
+The Electron project was born inside [Atom], GitHub's open-source text editor 
+built on web technologies. Electron was originally called `atom-shell`. Atom 
+was the first app to use it, but it didn't take long for folks to realize that 
+this magical Chromium + Node runtime could be used for all kinds of different 
+applications. When companies like Microsoft and Slack started to make use of
+`atom-shell`, it became clear that the project needed a new name.
+
+And so "Electron" was born. In early 2016, GitHub assembled a new team to focus
+specifically on Electron development and maintenance, apart from Atom. In the 
+time since, Electron has been adopted by thousands of app developers, and is now
+depenended on by many large companies, many of which have Electron teams of 
+their own.
+
+Supporting GitHub's Electron projects like Atom and [Desktop] is still a 
```

Comment:
Are we referring to **GitHub Desktop**?

Reply:
"Supporting GItHub's Electron projects..." so yes :smile: